lib/arriba/mime_type.rb

```diff
@@ -4,14 +4,51 @@ module MimeType
       'ai'    => 'application/postscript',
       'eps'   => 'application/postscript',
       'exe'   => 'application/octet-stream',
+      'pdf'   => 'application/pdf',
+      'xml'   => 'application/xml',
+      'swf'   => 'application/x-shockwave-flash',
+      # MS Office  - http://blogs.msdn.com/b/vsofficedeveloper/archive/2008/05/08/office-2007-open-xml-mime-types.aspx
       'doc'   => 'application/vnd.ms-word',
+      'dot'   => 'application/vnd.ms-word',
+      'docx'  => 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+      '.docm' => 'application/vnd.ms-word.document.macroEnabled.12',
+      '.dotm' => 'application/vnd.ms-word.template.macroEnabled.12',
       'xls'   => 'application/vnd.ms-excel',
+      '.xlt'  => 'application/vnd.ms-excel',
+      '.xla'  => 'application/vnd.ms-excel',
```

Comment:
The above have leading `.` characters -- is this intentional and/or does it still work with these in place?